src/app/app.module.ts

```diff
@@ -1,16 +1,38 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
```

Comment:
The frontend is developed in angular 2/4 which is in typescript, it has not been developed in javascript yet, so we had to use typescript for the frontend and for the backend, Node JS is available in typescript but that would probably require some changes to the structure and I am assuming there is little support for it. Also, the structure had the backend in javascript so I built on it.